### ISSUE TITLE
[BUG] Select-Menus don't work

### ISSUE THREAD

**Issue by HysMX** [NamVr/DiscordBot-Template#5]
Select Menus don't work, can be fixed by adding the following into bot.js

```javascript

/**********************************************************************/
// Registration of select-menus Interactions

/**
 * @type {String[]}
 * @description All Context Menu commands.
 */

 const selectMenus = fs.readdirSync("./interactions/select-menus");

 // Loop through all files and store select-menus in slashCommands collection.
 
 for (const folder of selectMenus) {
	 const files = fs
		 .readdirSync(`./interactions/select-menus/${folder}`)
		 .filter((file) => file.endsWith(".js"));
	 for (const file of files) {
		 const menus = require(`./interactions/select-menus/${folder}/${file}`);
		 const keyName = `${menus.id}`;
		 client.selectCommands.set(keyName, menus);
	 }
 }



```

Yes im too lazy to fork it and PR it

**Comment by NamVr** [comment]
Alright, thanks for letting me know! However your code was a bit inaccurate as I noticed it was mainly a _copy-pasta_ from context menus. Please note context menu is entirely a different thing.

**Comment by HysMX** [comment]
It is! But thats the only thing that made them work

**Comment by NamVr** [comment]
Fixed in e3bc1d2b1fa29ee19b4e2bc8d6b4f60da9f564f1, thanks

**Comment by HysMX** [comment]
Thanks to you!
